pom.xml

```diff
@@ -331,6 +331,13 @@
         </plugin>
       </plugins>
     </pluginManagement>
+
+    <resources>
```

Comment:
this can't go in src/main/resources?

Reply:
It could. It doesn't have to.

Reply:
I think, as much as we can, we should follow the standard directory format for maven : https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html

Given that resources is already defined, can you move the file into `src/main/resources`?

Reply:
it would remove extra configuration from pom.xml.